versions.properties

```diff
@@ -117,18 +86,25 @@ version.org.jboss.shrinkwrap..shrinkwrap-impl-base=1.2.6
 ##                                     # available=2.0.0-beta-1
 
 version.org.ops4j.pax.exam..pax-exam-container-native=4.13.1
+##                                        # available=4.13.2
+##                                        # available=4.13.3
+##                                        # available=4.13.4
+##                                        # available=4.13.5
 
 version.org.ops4j.pax.exam..pax-exam-link-mvn=4.13.5
 
 version.org.ops4j.pax.exam..pax-exam-testng=4.13.5
 
 version.org.ops4j.pax.url..pax-url-aether=2.6.12
+##                            # available=2.6.13
+##                            # available=2.6.14
 
-version.org.webjars..jquery=3.6.1
+version.org.webjars..jquery=3.7.1
 
-version.org.xmlunit..xmlunit-assertj=2.9.0
+version.org.xmlunit..xmlunit-assertj=2.9.1
+##                       # available=2.9.1
```

Comment:
I think it should be removed

Reply:
This cannot be removed because there are tests that are using this.

Please see https://github.com/testng-team/testng/blob/master/testng-core/src/test/java/test/reports/FailedReporterTest.java#L25

Reply:
I'm talking about the commented version, not the dependency.